### ISSUE TITLE
Last Accessibility Requests from Henry

### ISSUE THREAD

**Issue by adane** [Tizra/Tizra-Customer-Tracker#266]
Following is a thread relating to the last markup change requests from Henry, with a response from David D. at the top...

I think panel is from Foundation, unless its in the beta Reader, in which case it's a mistake…

We don't use that class internally in block structure, I don't think.

t-my should be in place shortly.

On Mon, Jul 8, 2013 at 1:15 PM, Henry Woodbury wrote:
Regarding the t- standard I noticed that a number of components are no longer using it. For example panel is "panel" instead of "t-panel". Is this from Foundation.css or just a carry-over from earlier code? Just curious.

---- On Mon, 08 Jul 2013 12:50:45 -0400 David Durand wrote ---- 

Shortening ID seems great. Given that I'm trying to have a rule of t- prefixes for tizra styles that can be "counted on" I think I will go for t-my. #t seems maybe just a little too short.

I think I've almost gotten through all my bugs in the beta reader stuff, so I'll try to get your markup upgrades in place quickly.

  -- David

---

David G. Durand

Online publishing unbound.

On Fri, Jul 5, 2013 at 1:32 PM, Henry Woodbury wrote:
Hi Abe, David

I've finished my accessibility pass through the standard template with a review of color contrast. To resolve several contrast issues I updated the site properties.

I have one more feature request.

For the list of colors defined under Properties, could we explicitly call out five more colors?

Link Default Color
Link Default Hover Color
Link Default Visited Color

Button Default Color
Button Default Hover Color

For the version of the publisher I've been working on, I assigned Main Color to both links and buttons, resulting in a garish (but compliant) site design. 

http://s70138590-preview.tizrapublisher.com/

I would recommend that "Main Color" is used for the text logo in the banner, and Main Headings (H1s). Otherwise, if "Main Color" is intended to be "Default Link Color", can we identify it as such?

The rest of my changes can proceed as follows:
1. David making the underlying changes to the platform I've already requested
2. Me updating the default style based on those updates
3. When this is done, updating the ARL template.

Just one more note. I started building the Stylesheet template in the Stylesheet tab. My idea is to give users a set of empty rules that they can choose to use for overrides (or not). It will integrate the #t-foundation ID we've discussed adding to the body tag. #t-foundation seems a little long. Since it will be added to every override rule, could we use something super short like #t or #my?

Thanks,

Henry

**Comment by daviddurand** [comment]
turns out `id="t-my"` has been on the body tag for a while.

So any stylesheet rule that you want to override, say  `div.foo a` can be overriden as `#t-my div.foo a`
